server/create-app.js

```diff
@@ -18,7 +18,8 @@ function createApp({assetHelper = require("./asset-helper"), publicFolder = "pub
         res.set('Cache-Control', 'public,max-age=31104000,s-maxage=31104000');
       }
       res.set('Vary', 'Accept-Encoding');
-    }
+    },
+    maxAge: '1h'
```

Comment:
hopefully this doesn't over ride the max-age set before. couldn't see how it gets merged in the docs.

Reply:
the tests are passing i'm guessing so shouldn't matter.